pkg/operator/ceph/controller/spec_test.go

```diff
@@ -125,6 +125,17 @@ func TestBuildAdminSocketCommand(t *testing.T) {
 	assert.Equal(t, "mon_status", command)
 }
 
+func buildDaemonReadinessCheckCommand(t *testing.T) {
+	c := getDaemonConfig(config.MgrType, "")
+
+	command := c.buildDaemonReadinessCheckCommand()
+	assert.Equal(t, "mgr_statusstatus", command)
+
+	c.daemonType = config.MonType
```

Comment:
We do not use the readiness probe for the mons

Reply:
Fixed